pkg/proxy/iptables/proxier.go

```diff
@@ -169,6 +169,8 @@ type Proxier struct {
 	haveReceivedServiceUpdate   bool // true once we've seen an OnServiceUpdate event
 	haveReceivedEndpointsUpdate bool // true once we've seen an OnEndpointsUpdate event
 	throttle                    flowcontrol.RateLimiter
+	timer                       *time.Timer
+	lastSync                    time.Time
```

Comment:
k, so I looked through this several times, and I see what you're saying, but I would move it out as a cleanup item.  

imho the proxy will likely be due for some overhaul in the mid-near-term due to the endpoints spamming.  Even with the filter changes, this is still 1:(N) broadcast and as we add a number of other HA components that will become C*N.  So refactor has to come.